MY EYES! - In which I revisit blinds again.

OneOneThreeEight opened the pull request:
Complete. Didn't actually want to buff flashbangs and flashpowder is actually so insanely powerful already that I refuse to buff it. Same deal with vampires, but I did so anyway

Essentially, buffs up handheld flashes and other flash sources to render people confused. Fixes an issue where mounted flashes and portable flashes didn't flick with "e_flash". Also adds confusion and dizzying effects if you happen to get blinded by any of the above. *The core part of the change is that some common forms of blinds will now dizzy and confuse.*

Primarily intended to be most useful for bad guy synthetics and malfunctioning AIs defending their core. Notably, flash rounds have also been buffed to have similar dazing effects, but are useless if you don't blind the person.

Being confused now has a 1/4th chance of making you step to a random tile instead of 1/5th.

OneOneThreeEight commented:
!wip

OneOneThreeEight commented:
!review

NonQueueingMatt commented on code/game/gamemodes/cult/runes/stun.dm:
```diff
@@ -20,6 +20,8 @@
 				C.stuttering = 1
 			if(is_rune)
 				C.Weaken(3)
+			C.make_dizzy(60)
```
This does not actually do anything. It has to be over 100 for dizziness to process. Imho the stun runes don't need dizzy.

NonQueueingMatt commented on code/game/gamemodes/cult/runes/stun.dm:
```diff
@@ -20,6 +20,8 @@
 			if(is_rune)
 				C.Weaken(3)
+			C.make_dizzy(60)
+			C.confused = rand(5,15)
```
This really should not be random with such a large interval.

NonQueueingMatt commented on code/game/objects/items/devices/flash.dm:
```diff
@@ -79,6 +79,8 @@
 		var/safety = M:eyecheck(TRUE)
 		if(safety <= 0)
 			flick("e_flash", M.flash)
+			M.make_dizzy(60)
```
This doesn't do anything and flashes shouldn't have it.

OneOneThreeEight commented:
Didn't know dizzy worked like that -- or rather didn't work at all. Changed either way.

Removed the randomness on request, might be better if it's more predictable actually.

NonQueueingMatt commented on code/modules/projectiles/projectile/energy.dm:
```diff
@@ -23,6 +23,8 @@
 	//blind adjacent people
 	for (var/mob/living/carbon/M in viewers(T, flash_range))
 		if(M.eyecheck() < FLASH_PROTECTION_MODERATE)
+			M.make_dizzy(60)
```
This doesn't do anything.
